src/org/apache/lucene/index/DocValuesFieldUpdates.m

```diff
@@ -17,9 +17,9 @@
 
 @implementation OrgApacheLuceneIndexDocValuesFieldUpdates
 
-+ (jint)PAGE_SIZE {
-  return OrgApacheLuceneIndexDocValuesFieldUpdates_PAGE_SIZE;
-}
+//+ (jint)PAGE_SIZE {
```

Comment:
Did you run into any compiler error with this method? Just want to make sure if it's a J2ObjC issue addressed in more recent source.